lms/djangoapps/course_blocks/transformers/start_date.py

```diff
@@ -85,4 +105,8 @@ def transform_block_filters(self, usage_info, block_structure):
             usage_info.course_key,
             now=now,
         )
+
+        if usage_info.include_has_scheduled_content:
+            block_structure = self._check_has_scheduled_content(block_structure, removal_condition)
```

Comment:
Is that the pattern with similar functions, to return the block structure? It just changes the structure in place, yeah? So we don't *need* the return, I'd have thought.

Reply:
I'm pulling that helper method's code back into this anyways, but you were right :)